test/test_integration_cli.py

```diff
@@ -118,40 +126,35 @@ def generate_policy_string(self):
         variable_3 = 'simple/special @#$%^&*(){{}}[]._+/{id}'.format(id=uuid.uuid4().hex)
 
         policy = \
-"""
-- !variable
-  id: {variable_1}
-- !variable
-  id: {variable_2}
-- !variable
-  id: {variable_3}
-"""
+            """
```

Comment:
I think the policy not indented. The """ is sensitive to indentation so the rendering of the actual policy will be impacted